src/index.js

```diff
@@ -66,102 +59,234 @@ class Radar {
     Cookie.setCookie(Cookie.DESCRIPTION, description);
   }
 
-  static trackOnce(callback) {
-    const publishableKey = Cookie.getCookie(Cookie.PUBLISHABLE_KEY);
-
-    if (!publishableKey) {
-      if (callback) {
-        callback(STATUS.ERROR_PUBLISHABLE_KEY);
-      }
+  static getLocation(callback) {
```

Comment:
Wonder if it still makes sense to expose this. It's just a wrapper around navigator?

Reply:
correct